.gitignore

```diff
@@ -14,6 +14,9 @@ notebooks/.ipynb_checkpoints
 notebooks/*/.ipynb_checkpoints
 .hypothesis/
 
+# Editor settings
+.vscode/*
```

Comment:
This seems to duplicate line 24 (although I hadn't included the wildcard) - maybe we should just delete line 24